### ISSUE TITLE
Placeholder Slot does not work

### ISSUE THREAD

**Issue by appinteractive** [egoist/vue-no-ssr#7]
Hey thanks for that lib at first! 

I'm using it with Nuxt.js and tried to display a loader in the mentioned placeholder.
Always getting ```[vue-no-ssr] You cannot use multiple child components```

```
<no-ssr>
  <b-table :items="items" />
  <div slot="placeholder">
    <h5>Loading...</h5>
  </div>
</no-ssr>
```

**Comment by snsxn** [comment]
Try using `template`and put it first:
```html
<no-ssr>
   <template slot="placeholder">
      <h5>Loading...</h5>
   </template>
   <b-table :items="items" />
</no-ssr>
```
It's working fine.

**Comment by appinteractive** [comment]
Still does not work. Getting the same warning but it seems like it only happens when the ssr content is replaced with the client content.

**Comment by egoist** [comment]
You example is correct, a bug indeed, I've fixed it in 1.0.0

**Comment by appinteractive** [comment]
Thx
